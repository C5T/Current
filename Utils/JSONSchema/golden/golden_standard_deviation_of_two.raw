{"Object":{"field_schema":[["x",{"Integer":{"sum":28,"sum_squares":140.0,"can_be_unsigned":true,"can_be_microseconds":false,"instances":7,"nulls":1}}],["comment",{"String":{"values":{"standard deviation must be two":1},"counters":[[1,"standard deviation must be two"]],"instances":1,"nulls":1}}]],"field_index":{"comment":1,"x":0},"instances":8,"nulls":0}}