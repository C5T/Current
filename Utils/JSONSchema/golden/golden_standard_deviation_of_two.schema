// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(comment, Optional<std::string>);
  CURRENT_FIELD(x, Optional<uint64_t>);
};

using Schema = Schema_Object;
