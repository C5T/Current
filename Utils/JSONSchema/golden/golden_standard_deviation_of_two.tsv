Field	Type	Set	Unset/Null	Values	Details
Schema	Object	8	0	2 fields	comment, x
Schema.comment	String	1	1	1 distinct value, "standard deviation must be two":1
Schema.x	Integer	7	1	Mean 4, StdDev 2
